tools/triage/ticket_search/cli/search_for_tickets.py

```diff
@@ -0,0 +1,109 @@
+"""Module for command line utility to fetch a single triage ticket"""
+import argparse
+import logging
+import os
+import sys
+
+from command import Command
+from workflow_data import WorkflowData
+
+from tools.jira_client import JiraClientFactory
+from tools.triage.ticket_search import (
+    JiraAttachmentDownloader,
+    TicketFetcher,
+    TicketParser,
+    TicketQuery,
+)
+from tools.triage.ticket_search.directory_cache import DirectoryCache
+
+
+class SearchForTickets(Command):
+    """Responsible for fetching a single ticket for a triage issue"""
+
+    def __init__(self):
+        """Constructor"""
+        self.args = []
+        self.unknown_args = []
+        self.logger = logging.getLogger(__name__)
+
+    def get_command_key(self) -> str:
+        return "search_for_tickets"
+
+    def get_friendly_name(self) -> str:
+        return "Search for tickets"
+
+    def run(self, workflow_data: "WorkflowData") -> "WorkflowData":
+        """This runs the command"""
+        # Set up the ticket fetcher
+        self.workflow_data = workflow_data
+        # Adding dry_run_stream forces the client to be "read only"
```

Comment:
This is misleading though, even if there is this comment....
I understand introducing now a read-only client would be out of scope and not trivial though.